### PR TITLE
Add dotnet/performance to repositories.props

### PR DESCRIPTION
**Pull request by tannergooding**
This resolves https://github.com/dotnet/source-indexer/issues/63. I also fixed the repo url for msbuild while I was here.

### CODE REVIEW

**Comment by tannergooding** [comment]
CC. @DrewScoggins, @AndyAyersMS, @kunalspathak, @adamsitnik, @alexperovich

**Comment by tannergooding** [comment]
@alexperovich, is there anything blocking this or can it be merged?

I was just waiting on sign-off.

**Review by alexperovich — APPROVED** [review]

**Comment by alexperovich** [comment]
This is good to merge. There is another issue that I am tracking down so this will take a bit to be reflected on the website.

**Pull request merged**
